flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/StreamSQLExample.java

```diff
@@ -42,9 +47,24 @@
 
 	public static void main(String[] args) throws Exception {
 
+		final ParameterTool params = ParameterTool.fromArgs(args);
+		String planner = params.has("planner") ? params.get("planner") : "flink";
+
 		// set up execution environment
 		StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
-		StreamTableEnvironment tEnv = StreamTableEnvironment.create(env);
+		StreamTableEnvironment tEnv;
+		if (Objects.equals(planner, "blink")) {	// use blink planner in streaming mode
+			EnvironmentSettings settings = EnvironmentSettings.newInstance()
+				.useBlinkPlanner()
+				.inStreamingMode()
+				.build();
+			tEnv = StreamTableEnvironment.create(env, settings);
+		} else if (Objects.equals(planner, "flink")){	// use flink planner in streaming mode
+			tEnv = StreamTableEnvironment.create(env);
+		} else {
+			System.err.println("The planner is incorrect. Please use --planner to specify planner type.");
```

Comment:
Could you explain it more clearly in this error message that you have use either `flink` planner or `blink` planner?

Reply:
The user will not have such knowledge when they are trying the examples without looking into the codes.